be/src/olap/lru_cache.h

```diff
@@ -300,6 +300,7 @@ class LRUCache {
     Cache::Handle* lookup(const CacheKey& key, uint32_t hash);
     void release(Cache::Handle* handle);
     void erase(const CacheKey& key, uint32_t hash);
+    void prune_one(LRUHandle* old);
```

Comment:
should be a private method, named `_prune_one`

Reply:
Ok, pay attention later